public/data/CanisCoder.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Anand Kumar Maurya",
+  "bio": "Open Source Enthusiast | MERN Stack Developer | Competitive Coder | CSE student at @AITM ",
```

There's an extra space here.
```suggestion
  "bio": "Open Source Enthusiast | MERN Stack Developer | Competitive Coder | CSE student at @AITM",
```

---

I think the GitHub format action will take care of it!